src/util/psbt/map/input.rs

```diff
@@ -172,7 +172,8 @@ impl FromStr for PsbtSigHashType {
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         let ecdsa_sighash_ty = EcdsaSigHashType::from_str(s);
-        let num = u32::from_str_radix(s.trim_start_matches("0x"), 16);
+        // TODO: Revert commit once MSRV >= 1.30 to use non-deprecated `trim_start_matches`
+        let num = u32::from_str_radix(s.trim_left_matches("0x"), 16);
```

Comment:
I get what you are doing with the `REVERT-ME` in commit brief message but I get the feeling that, since this is the only use of this, that the code will just get fixed and the commit will be left in there. I also just don't like the idea of having a commit with `REVERT-ME` in the brief message, it is going to scream at any dev that sees it making them check whats going on.

Reply:
So what is the suggested way of doing this?